protocol/0057-TRAN-transfers.md

```diff
@@ -67,6 +67,9 @@ A recurring transfers needs to contain this specific information:
 - start epoch: at the end of this epoch the first recurring transfer will be made between
 - end epoch (optional): at the end of this epoch the last recurring transfer will be made between, optional. If not specified the transfer run until cancelled (by its creator or by the network as described below).
 - factor, decimal > 0.0 (a factor used with the amount specified for the transfer).
+- transfer interval: number of epochs between transfers, i.e. when 4, funds will be transferred every 4 epochs with the first transfer occurring 4 epochs after the transaction is processed. Must be an integer strictly greater than `0` and less than `100` (this ceiling is for performance reasons and matches the limit on the `window_length` field).
```

Comment:
I think it would be better to make this optional for backward compatibility and no value is interpreted as 1.

Reply:
also, I'm assuming this goes for both recurring and governance recurring transfers but should probably be made explicit here or on the ACs.

Reply:
how does transfer_interval play with window size, i.e. unless the window size is greater than or equal to the transfer interval then that means there are "dead epochs" which don't count towards the reward. E.g. window size is 1 and transfer_interval is 7, then we're only looking at the last epoch and the previous 6 are not counting towards anything (but the cap).

Reply:
Agreed on making the field optional and yes it should go for governance transfers as well. Will make updates for both of those.

For how it works with respect to window size, yes in the example you give I there will be dead epochs which don't count towards each parties score. Most likely users are going to setup rewards where `transfer_interval=window_size` (i.e. pay out every n epochs assessing performance over the last n epochs). 

I can't think of a reason why setting `transfer_interval!=window_size` would cause issues so my 2-cents is allow it, but in the docs make it clear how these fields relate to each other (can make some spec clarifications as well if needed).

Reply:
I don't think we shouldn't allow it but I think transfer_interval<window_size makes no sense as a configuration :)

Reply:
maybe makes-no-sense is too harsh though, anyhow shouldn't be a problem.

Reply:
> I don't think we shouldn't allow it but I think transfer_interval>window_size makes no sense as a configuration :)

Maybe a recurring weekend trading competition? (transfer_interval=7, window_size=2). I'm playing devils advocate though, I agree it's not the most sensible setup.